com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+                "--with-libpng=builtin",
+                "--with-libtiff=builtin",
+                "--with-libjpeg=builtin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.1/wxWidgets-3.1.1.tar.bz2",
+                    "sha256": "c925dfe17e8f8b09eb7ea9bfdcfcc13696a3e14e92750effd839f5e10726159e"
+                }
+            ]
+        },
+        {
+            "name": "freeimage",
+            "buildsystem": "autotools",
+            "no-autogen": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/freeimage/FreeImage3180.zip",
+                    "sha256": "f41379682f9ada94ea7b34fe86bf9ee00935a3147be41b6569c9605a53e438fd"
+                },
+                {
+                    "type": "patch",
+                    "path": "freeimage-fix-install.patch",
+                    "strip-components": 0
+                }
+            ]
+        },
+        {
+            "name": "pandoc",
+            "only-arches": [
+                "x86_64"
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp bin/pandoc /app/bin/pandoc",
+                "cp bin/pandoc-citeproc /app/bin/pandoc-citeproc"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/jgm/pandoc/releases/download/2.2/pandoc-2.2-linux.tar.gz",
+                    "sha256": "06ecd882e42ef9b7390b1c82e1e71b3ea48679181289b9b810a8797825bed8ed"
+                }
+            ]
+        },
+        {
+            "name": "trenchbroom",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_CXX_FLAGS=-Werror",
+                "-DwxWidgets_PREFIX=/app",
+                "-DOpenGL_GL_PREFERENCE=GLVND"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/kduske/TrenchBroom.git",
+                    "tag": "v2019.2",
+                    "commit": "67d4016c0b8e8a6cc6cbb11eb6b9ddf2779548ed"
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "TB_DEV_MODE=1 trenchbroom $@"
+                    ],
+                    "dest-filename": "trenchbroom.sh"
+                },
+                {
+                    "type": "file",
+                    "path": "com.kristianduske.TrenchBroom.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/kduske/TrenchBroom/v2019.2/app/resources/linux/trenchbroom.desktop",
```

Comment:
Not needed. The first source (git) already has a desktop file.

Reply:
Yeah, that's true. You see... I inherited this file from another person who wanted to build the Flatpak before but wasn't able to make it happen. I offered to step in and made the necessary minimal changes for it to build and work and didn't think much about stuff as this. I'll clear those lines. Thanks for pointing out these things.